assets/scss/user-activity.scss

```diff
@@ -0,0 +1,20 @@
+h2 {
+  font-size: 19px;
+}
+
+table.audit tr {
+  th:first-child {
+    white-space: nowrap;
+  }
+  td {
+    text-align: right;
+  }
+  .weekend {
+    display: none;
+    background-color: lightblue;
```

Comment:
In the future, we should probably change this to use the GovUK palette. But for now, meh.